### DISCUSSION TITLE
Resizable left spaces list

### DISCUSSION THREAD

**Discussion by tornellihenrique** [GitbookIO/community#389] [Feature Requests]
We are currently trying to use GitBook as the new internal wiki for the company. However, due to the way we organize spaces into subgroups, it has become very difficult to work, and some strange scroll bars are appearing.

We would like to explore the possibility of adding a resizable feature to the listing menu, as this would be very helpful.
It's worth mentioning that we are currently on the Plus plan.

Here is a portion of our spaces listing (I am not allowed to fully display it):
![image](https://github.com/GitbookIO/community/assets/47005931/1dcb87cc-22a9-4234-84fd-502ab5ec6bec)

We envisioned something similar to this layout from GitHub:
![a84c7835540b755e4b59001546a2ac7b](https://github.com/GitbookIO/community/assets/47005931/ce817ced-da0c-472b-8cff-f816e8b0b7fb)

Thank you in advance!

**Comment by thebiglabasky** [comment]
Hi,
We're currently working on improving the sidebar which will at least remove the double scroll. There are discussions to further improve the management of the screen's real estate depending on resolutions so watch this space :)

**Reply by NavithGames** [reply]
thanks!!!

**Reply by thebiglabasky** [reply]
The double scroll fix was shipped recently, so you should have a bit more space.
We'll keep this opened until we figure out a resizable sidebar

**Reply by NavithGames** [reply]
I'm waiting for a resizable sidebar to be implemented. 
Thank you for your work   :)
